core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -688,7 +682,8 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     OneTimeRevealSpecificMapTiles("Reveal up to [amount/'all'] [tileFilter] within a [amount] tile radius", UniqueTarget.Ruins),
     OneTimeRevealCrudeMap("From a randomly chosen tile [amount] tiles away from the ruins, reveal tiles up to [amount] tiles away with [amount]% chance", UniqueTarget.Ruins),
     OneTimeTriggerVoting("Triggers voting for the Diplomatic Victory", UniqueTarget.Triggerable),  // used in Building
-    OneTimeGlobalAlert("Triggers the following global alert: [comment]", UniqueTarget.Triggerable), // used in Policy
+    OneTimeGlobalAlert("Triggers the following global alert: [comment]", UniqueTarget.Policy), // used in Policy
+    OneTimeGlobalSpiesWhenEnteringEra("Every major Civilization gains a spy when entering this era", UniqueTarget.Era),
```

Comment:
It might be worth rewording to something like `"Every major Civilization gains a spy as soon as the first civilization enters this era"` since otherwise it sounds like each civ gains a spy as soon as _they_ enter the era rather than what actually happens in code which is all civs gain a spy.